WizelineLabs/project-lab
# adding typesense and instantsearch

# What does this PR do?
--Add Typensense 
--Add Instatnsearch for improbing searching
--Modifications on the header

# Where should the reviewer start?

on the root folder:
--projects_index.tsx
--intershipProjects_index.tsx

new typsense folder


# How should this be manually tested?

to test it locally you have to follow this:

1. install docker if you dont have it

2. add this to your `.env`:

```
TYPESENSE_HOST=localhost
TYPESENSE_PORT=8108
TYPESENSE_PROTOCOL=http
TYPESENSE_API_KEY=123
TYPESENSE_ADMIN_API_KEY=123
REACT_APP_TYPESENSE_SEARCH_ONLY_API_KEY=123
REACT_APP_TYPESENSE_HOST=localhost
REACT_APP_TYPESENSE_PORT=8108
REACT_APP_TYPESENSE_PROTOCOL=http
```

3. start the server with the command:
`npm run startypesense
`
and check on your docker if the image is done, then you will have to start it

4. fill the server with your database using:
   `npm run populate`

# Any background context you want to provide?



# What are the relevant tickets?

[](https://github.com/orgs/WizelineLabs/projects/3/views/1?pane=issue&itemId=55573749)

# Screenshots



# Questions



# Checklist
